eahub/base/utils.py

```diff
@@ -13,6 +14,10 @@ def user_display(user):
     return profile.name
 
 
+def get_feedback_url(request):
+    return f"https://feedback.{get_current_site(request).domain}"
```

Comment:
thank you for helping here!

AFAIK sadly get_current_site + request isn't as reliable as I wish it were, I don't remember the details, but I believe the request might contain not enough data in few cases and throw errors. Not 100% sure about it thought, hence just warning.

In the end I had to rely on divio.com env variable as `DOMAIN` or add it manually on non-divio/heroku projects